### ISSUE TITLE
weather execution error

### ISSUE THREAD

**Issue by simonbcn** [oh-my-fish/oh-my-fish#214]
fish, version 2.2.0-490-g994aba0

```
weather                                                                  02:49
Weather for Terrassa, Spain

Temperature: 9.10 °C (48.38 °F)
   Humidity: 100%
 Cloudiness: broken clouds
   Pressure: 1031 hpa
       Wind: from NW (315°) at 1.03 m/s

5-day forecast
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
error: strftime is not defined
.dt | strftime("%d/%m")      1 compile error
contains: Key not specified
  Temperature: ▁▁▁▃▃▃▃▄▃▄▅▆▆▃▃▂▂▂▃▅▅▄▄▅▅▅▅███▅▄▃▃▃▅▅▄▄▄
               

Precipitation: ▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁
```

**Comment by unusedPhD** [comment]
had the same issue running version 1.4-2.1 of jq from ubuntu default repo.  Issue was solved by downloading v1.5 and updating the binary.

**Comment by simonbcn** [comment]
Thanks, it was that. The problem is the lack of documentation about this packages.
